scripts/preload_models.py

```diff
@@ -36,6 +40,19 @@
 transformer = CLIPTextModel.from_pretrained(version)
 print('...success')
 
+
+print('preloading Stable Diffusion model file...', end='')
```

Comment:
For some reason, the message "preloading Stable Diffusion model file" doesn't appear for me. It goes right into Downloading without explaining what is being downloaded:

~~~
$ python scripts/preload_models.py 
preloading bert tokenizer......success
preloading Kornia requirements......success
preloading CLIP model......success
Downloading: 100%|██████████████████████████████
~~~

If the file is already cached, then I see this, which is fine:
~~~
preloading bert tokenizer......success
preloading Kornia requirements......success
preloading CLIP model......success
preloading Stable Diffusion model file......success
Loading models from RealESRGAN and facexlib......success
Loading models from GFPGAN
preloading CodeFormer model file......success
~~~

Reply:
hmm, I wouldn't be surprised if tqdm goes back to the beginning of the line and overwrites it.

well, the _easy_ thing to do would be to take off that `end=''` so the start message is always on its own line. Would that be okay, or do we need something slicker?